<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlustusDialogi</class>
 <widget class="QDialog" name="AlustusDialogi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pelin alustus</string>
  </property>
  <widget class="QPushButton" name="OK_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Peruuta_Button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="AlustenLKM_spin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QSlider" name="AlustenLKM_slide">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Alusten lukumäärä</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pelaajan nimi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PelaajanNimi_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Pelaaaja 1</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AlustenLKM_slide</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AlustenLKM_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AlustenLKM_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AlustenLKM_slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
